### PR TITLE
zh: update docs about shard DDL lock

### PR DESCRIPTION
**Pull request by csuzhangxc**
### What is changed, added or deleted? (Required)

update docs about shard DDL lock because we have re-implemented it in v2.0.

### Which DM version(s) do your changes apply to? (Required)

- [x] master (the latest development version, including v2.0 changes for now)
- [ ] v1.0 (TiDB DM 1.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [x] Have version specific changes 
- [ ] Might cause conflicts

### CODE REVIEW

File: zh/manually-handling-sharding-ddl-locks.md [hunk 1]
```diff
@@ -341,45 +266,46 @@ MySQL 及 DM 操作与处理流程如下：
...
 
-DM-worker-2 重启后，将尝试重新同步重启前已经在等待的 DDL lock。此时将在 DM-master 上创建一个新的 lock，并且该 DM-worker 将成为 lock 的 owner（其他 DM-worker 此时已经执行或跳过 DDL 操作并在进行后续同步）。
+`mysql-replica-02` 对应的数据迁移子任务恢复后，将在 DM-master 上创建一个新的 lock，但其他 MySQL source 此时已经执行或跳过 DDL 操作并在进行后续同步。
 
 处理流程如下：
 
 1. 使用 `show-ddl-locks` 确认 DM-master 上存在该 DDL 操作对应的 lock。
 
-    应该仅有该重启的 DM-worker（`127.0.0.1:8263`）处于 `syned` 状态：
+    应该仅有 `mysql-replica-02` 处于 `syned` 状态：
```
Comment (GMHDBJD) [inline zh/manually-handling-sharding-ddl-locks.md:308]:
```suggestion
    应该仅有 `mysql-replica-02` 处于 `synced` 状态：
```

File: zh/manually-handling-sharding-ddl-locks.md [hunk 2]
```diff
@@ -285,21 +223,8 @@ MySQL 及 DM 操作与处理流程如下：
...
-                },
-                {
-                    "result": false,
-                    "worker": "127.0.0.1:8263",
-                    "msg": "rpc error: code = Unavailable desc = all SubConns are in TransientFailure, latest connection error: connection error: desc = \"transport: Error while dialing dial tcp 127.0.0.1:8263: connect: connection refused\""
-                }
-            ]
-        }
+            "result": true,
+           "msg": ""
```
Comment (GMHDBJD) [inline zh/manually-handling-sharding-ddl-locks.md:227]:
```suggestion
            "msg": ""
```

File: zh/manually-handling-sharding-ddl-locks.md [hunk 3]
```diff
@@ -72,23 +73,24 @@ show-ddl-locks test
 
 该命令用于主动请求 DM-master 解除指定的 DDL lock，包括的操作：请求 owner 执行 DDL 操作，请求其他非 owner 的 DM-worker 跳过 DDL 操作，移除 DM-master 上的 lock 信息。
 
+> **注意：**
+>
+> `unlock-ddl-lock` 当前仅对悲观协调模式（`pessimistic`）下产生的 lock 有效。
```
Comment (ireneontheway) [inline zh/manually-handling-sharding-ddl-locks.md:78]:
```suggestion
> `unlock-ddl-lock` 当前仅对悲观协调模式(`pessimistic`)下产生的 lock 有效。
```
> 括号里全为英文时使用半角括号，并在括号前后各空一个半角空格，括号和括号内的英文之间不需要空格。 from[ 中文技术文档规范](https://docs.google.com/document/d/1b6ZhZD33OoM8AacpKksGGSuxJWReLkNnSt8eSc1kTXc/edit#)

File: zh/manually-handling-sharding-ddl-locks.md [hunk 4]
```diff
@@ -110,85 +112,22 @@ unlock-ddl-lock test-`shard_db`.`shard_table`
...
-目前，使用 `unlock-ddl-lock`/`break-ddl-lock` 命令仅支持处理以下三种 sharding DDL lock 异常情况。
+目前，使用 `unlock-ddl-lock` 命令仅支持处理以下两种 sharding DDL lock 异常情况。
 
-### 场景一：部分 DM-worker 下线
+### 场景一：部分 MySQL source 被移除
 
 #### Lock 异常原因
 
-在 DM-master 尝试自动 unlock sharding DDL lock 之前，需要等待所有 DM-worker 的 sharding DDL events 全部到达（详见[分库分表合并同步原理](feature-shard-merge-pessimistic.md#实现原理)）。如果 sharding DDL 已经在同步过程中，且有部分 DM-worker 下线，并且不再计划重启它们（按业务需求移除了这部分 DM-worker），则会由于永远无法等齐所有的 DDL 而造成 lock 无法自动 unlock。
+在 DM-master 尝试自动 unlock sharding DDL lock 之前，需要等待所有 MySQL source 的 sharding DDL events 全部到达（详见[分库分表合并同步原理](feature-shard-merge-pessimistic.md#实现原理)）。如果 sharding DDL 已经在同步过程中，且有部分 MySQL source 被移除，并且不再计划重新加载它们（按业务需求移除了这部分 MySQL source），则会由于永远无法等齐所有的 DDL 而造成 lock 无法自动 unlock。
```
Comment (ireneontheway) [inline zh/manually-handling-sharding-ddl-locks.md:126]:
```suggestion
在 DM-master 尝试自动 unlock sharding DDL lock 之前，需要等待所有 MySQL source 的 sharding DDL events 全部到达（详见[分库分表合并同步原理](feature-shard-merge-pessimistic.md#实现原理)）。如果 sharding DDL 已经在同步过程中，同时有部分 MySQL source 被移除，且不再计划重新加载它们（按业务需求移除了这部分 MySQL source），则会由于永远无法等齐所有的 DDL 而造成 lock 无法自动 unlock。
```

File: zh/manually-handling-sharding-ddl-locks.md [hunk 5]
```diff
@@ -49,19 +49,20 @@ show-ddl-locks test
...
     "msg": "",                                             # 查询 lock 操作失败时的原因或其它描述信息（如不存在任务 lock）
-    "locks": [                                             # DM-master 上存在的 lock 信息列表
+    "locks": [                                             # 当前存在的 lock 信息列表
         {
             "ID": "test-`shard_db`.`shard_table`",         # lock 的 ID 标识，当前由任务名与 DDL 对应的 schema/table 信息组成
             "task": "test",                                # lock 所属的任务名
-            "owner": "127.0.0.1:8262",                     # lock 的 owner（第一个遇到该 DDL 的 DM-worker）
-            "DDLs": [                                      # lock 对应的 DDL 列表
+            "mode": "pessimistic"                          # shard DDL 协调模式，可为悲观模式 "pessimistic" 或乐观模式 "optimistic"
+            "owner": "mysql-replica-01",                     # lock 的 owner（在悲观模式时为第一个遇到该 DDL 的 source ID, 在乐观模式时总为空）
```
[the remark this answers is not in the document]
Comment (csuzhangxc) [reply zh/manually-handling-sharding-ddl-locks.md]: 在 https://github.com/pingcap/docs-dm/pull/241/commits/e98ff103bd3095a5e56f3fdf7ef95580f426f146 改了，但和前面的统一，结尾没有加 `。`

File: zh/manually-handling-sharding-ddl-locks.md (hunk 3 above)
Comment (csuzhangxc) [reply zh/manually-handling-sharding-ddl-locks.md]: 👍  在 https://github.com/pingcap/docs-dm/pull/241/commits/e98ff103bd3095a5e56f3fdf7ef95580f426f146 按规范改了

File: zh/manually-handling-sharding-ddl-locks.md (hunk 4 above)
Comment (csuzhangxc) [reply zh/manually-handling-sharding-ddl-locks.md]: 在 https://github.com/pingcap/docs-dm/pull/241/commits/e98ff103bd3095a5e56f3fdf7ef95580f426f146 改了

**Pull request merged**